traffic_ops/app/db/migrations/20170221190747_add_reval_pending.sql

```diff
@@ -0,0 +1,26 @@
+/*
+
```

Comment:
this migration file needs a later timestamp on the name. see existing migrations - https://github.com/apache/incubator-trafficcontrol/tree/master/traffic_ops/app/db/migrations

it needs to be timestamped later than the add_client_steering migration...